code/modules/admin/verbs/onlyone.dm

```diff
@@ -48,6 +47,10 @@
 		W.update_label(H.real_name)
 		H.equip_to_slot_or_del(W, slot_wear_id)
 
+		var/obj/item/weapon/claymore/highlander/H1 = new(H)
+		H.put_in_hands(H1)
+		H1.pickup(H)

```

Comment:
Items have pickup as a proc that refers to the mob picking them up. It's called manually because Byond.